test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -23,13 +23,51 @@
 using namespace std;
 using namespace yul::test::yul_fuzzer;
 
+std::string const yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
+{
+	std::string tmp{_hexBytes};
+	if (!tmp.empty())
+	{
+		tmp.erase(std::remove_if(tmp.begin(), tmp.end(), [](char c) {
+			return !std::isxdigit(c);
+		}), tmp.end());
+		tmp = tmp.substr(0, 64);
+	}
+	// We need this awkward if case because tmp can also be null post erase call above
+	if (tmp.empty())
+		tmp = "1";
+	return tmp;
+}
+
+std::string const yul::test::yul_fuzzer::createAlphaNum(std::string const& _strBytes)
+{
+	std::string tmp{_strBytes};
+	if (!tmp.empty())
+	{
+		tmp.erase(std::remove_if(tmp.begin(), tmp.end(), [](char c) {
+			return !(std::isalpha(c) || std::isdigit(c));
+		}), tmp.end());
+		tmp = tmp.substr(0, 32);
+	}
+	// We need this awkward if case because tmp can also be null post erase call above
+	if (tmp.empty())
+		tmp = "empty";
+	return tmp;
+}
+
 ostream& yul::test::yul_fuzzer::operator<<(ostream& _os, Literal const& _x)
 {
 	switch (_x.literal_oneof_case())
 	{
 		case Literal::kIntval:
 			_os << _x.intval();
 			break;
+		case Literal::kHexval:
+			_os << "0x" << createHex(_x.hexval());
+			break;
+		case Literal::kStrval:
+			_os << "\"" << createAlphaNum(_x.strval()) << "\"";
```

Comment:
Do we really have support for string literals?

Reply:
This works :)

```
$ echo {let x := add("hello", "world")} | ./solc/solc --strict-assembly -
```

iiuc, string literals are another way to encode literal values